doc/releases/changelog-0.20.0.md

```diff
@@ -642,20 +660,15 @@
 * Improves the Developer's Guide Testing document.
   [(#1896)](https://github.com/PennyLaneAI/pennylane/pull/1896)
 
-* Add documentation example for AngleEmbedding, BasisEmbedding, StronglyEntanglingLayers, SqueezingEmbedding, DisplacementEmbedding,
-  MottonenStatePreparation and Interferometer.
+* Added documentation examples for `AngleEmbedding`, `BasisEmbedding`, `StronglyEntanglingLayers`,
+  `SqueezingEmbedding`, `DisplacementEmbedding`, `MottonenStatePreparation` and `Interferometer`.
   [(#1910)](https://github.com/PennyLaneAI/pennylane/pull/1910)
   [(#1908)](https://github.com/PennyLaneAI/pennylane/pull/1908)
   [(#1912)](https://github.com/PennyLaneAI/pennylane/pull/1912)
   [(#1920)](https://github.com/PennyLaneAI/pennylane/pull/1920)
   [(#1936)](https://github.com/PennyLaneAI/pennylane/pull/1936)
   [(#1937)](https://github.com/PennyLaneAI/pennylane/pull/1937)
 
-* QueueContext was not empty when importing `pennylane`.
-
-* Fixed circuit drawing problem with Interferometer and CVNeuralNet.
-  [(#1953)](https://github.com/PennyLaneAI/pennylane/issues/1953)
-  
 <h3>Contributors</h3>
 
 This release contains contributions from (in alphabetical order):
```

Comment:
Just double checking that no-one is missing here! For example, I think Utkarsh is missing

Reply:
We should also add Ali Asadi, Chae-Yeun Park, Isidor Schoch

Reply:
And Lee!

Reply:
:+1: